Tools/Scripts/pgo-profile-merge

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python3
+
+import argparse
+import subprocess
+import os
+
+def pad(string, maxLength):
```

Comment:
It doesn't look like `pad` and `shortened` are being used in this script - are they necessary to have here?